### PR TITLE
AI-powered search tutorial

### PR DESCRIPTION
**Pull request by guimachiavelli**

### CODE REVIEW

**Review by macraig — APPROVED** [review]
Super clear and easy to read 🥇

**Review by dureuill — APPROVED** [review]
Looking good and to the point, thank you @guimachiavelli 👍

File: learn/search/getting_started_with_ai_search.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  }'
+```
+
+Meilisearch will return a mix of semantic and keyword matches, prioritizing results that match the query's meaning and context. If you want Meilisearch to return more results based on the meaning and context of a search, set `semanticRatio` to a value greater than `0.5`. Setting `semanticRatio` to a value lower than `0.5`, instead, will return more keyword matches.
+
+## Conclusion
+
+You have seen how to setup and perform AI-powered searches with Meilisearch and OpenAI. For more in-depth information, consult the reference for embedders and the `hybrid` search parameter.
+
+AI-powered is an experimental Meilisearch feature and is undergoing active development—[join the discussion on GitHub](https://github.com/orgs/meilisearch/discussions/621).
```
[the remark this answers is not in the document]
Comment (guimachiavelli) [reply learn/search/getting_started_with_ai_search.mdx:106]:
```suggestion
AI-powered is an experimental Meilisearch feature and is undergoing active development—[join the discussion on GitHub](https://github.com/orgs/meilisearch/discussions/677).
```

File: learn/search/getting_started_with_ai_search.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "vectorStore": true
+  }'
+```
+
+## Generate vector embeddings with OpenAI
+
+Next, you must generate vector embeddings for all documents in your dataset. Embeddings are mathematical representations of the meanings of words and sentences in your documents. Meilisearch relies on external providers to generate these embeddings. Use OpenAI for this tutorial.
```
[the remark this answers is not in the document]
Comment (guimachiavelli) [reply learn/search/getting_started_with_ai_search.mdx:51]: Definitely want to keep it simple for this one.

**Pull request merged**